api/v1alpha08/sonataflowplatform_types.go

```diff
@@ -111,6 +122,8 @@ type SonataFlowPlatformStatus struct {
 	// Triggers list of triggers created for the SonataFlowPlatform
 	//+operator-sdk:csv:customresourcedefinitions:type=status,displayName="triggers"
 	Triggers []SonataFlowPlatformTriggerRef `json:"triggers,omitempty"`
+	//+operator-sdk:csv:customresourcedefinitions:type=status,displayName="dbMigrationStatus"
+	SonataFlowPlatformDBMigrationStatus *SonataFlowPlatformDBMigrationStatus `json:"sonataFlowPlatformDBMigrationStatus,omitempty"`
```

Comment:
We will refactor this to the new Status API anyway, but if your type has only one field, this is unnecessary unless you add `message` and `reason`.